src/backend/TrafficCourts/Staff.Service/Authentication/Resources.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace TrafficCourts.Staff.Service.Authentication;
+
+/// <summary>
+/// Defines all the resources that are secured in the application.
+/// </summary>
+public static class Resources
+{
+    public const string Dispute = "dispute";
+    public const string Ticket = "ticket";
+    public const string TicketInbox = "ticket-inbox";
+    public const string JudicialJustice = "judicial-justice";
+    public const string JudicialJusticeDispute = "judicial-justice-dispute";
+}
```

Comment:
These resources are different than the ones defined on the confluence site: https://justice.gov.bc.ca/wiki/pages/viewpage.action?pageId=345243986#JJ&StaffWorkbenchRoles-Resource/ScopeMapping

Reply:
Resolved. I will leave it up to the implementor to use the correct values.